.github/workflows/tests.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           enable-cache: true
       - run: uv python pin ${{ matrix.python-version }}
       - uses: hynek/build-and-inspect-python-package@v2
-      - run: uv sync --python-preference=only-managed
+      - run: uv sync --python-preference=only-managed --all-extras
```

Comment:
So this change is suboptimal because what will happen for local users is they will run the test but won't have the `[batch]` extra because local users just run `uv sync`.

What we should do here is in `tool.uv`:

```toml
[dependency-groups]
dev = [
    ...
    "paper-qa[datasets,batch,ldp,typing,zotero,local]",
    ...
]
```

And then all use cases will be covered. No need for the `--all-extras`

Reply:
We have a similar declaration in `project.toml`, but I found no `tool.uv`. Are they the same thing?

Reply:
Ah crap, yeah it's `[dependency-groups]` now, yeah that was changed recently